test/simulator/main/org/apache/cassandra/simulator/paxos/AccordSimulationRunner.java

```diff
@@ -60,19 +60,27 @@ public static class Help extends HelpCommand<AccordClusterSimulation.Builder> {}
     /**
      * See {@link org.apache.cassandra.simulator} package info for execution tips
      */
-    public static void main(String[] args) throws IOException
+    public static void main(String... args) throws IOException
     {
         AccordClusterSimulation.Builder builder = new AccordClusterSimulation.Builder();
         builder.unique(uniqueNum.getAndIncrement());
 
-        Cli.<ICommand<AccordClusterSimulation.Builder>>builder("accord")
-           .withCommand(Run.class)
-           .withCommand(Reconcile.class)
-           .withCommand(Record.class)
-           .withCommand(Help.class)
-           .withDefaultCommand(Help.class)
-           .build()
-           .parse(args)
-           .run(builder);
+        try
+        {
+            Cli.<ICommand<AccordClusterSimulation.Builder>>builder("accord")
+               .withCommand(Run.class)
+               .withCommand(Reconcile.class)
+               .withCommand(Record.class)
+               .withCommand(Help.class)
+               .withDefaultCommand(Help.class)
+               .build()
+               .parse(args)
+               .run(builder);
+            System.exit(0);
+        }
+        catch (Throwable t)
+        {
+            System.exit(1);
```

Comment:
feel that all simulator runners need this... when an error is throw the `main` thread dies, but not the background non-daemon threads...

Reply:
Ok this is a problem for current tests: https://app.circleci.com/pipelines/github/dcapwell/cassandra/1666/workflows/7dd47b41-c2ed-4074-a9cd-6f9a8a72a4cc/jobs/13823

Rather than do system exit we need to throw... but at least when I was testing we had daemon threads which blocked the JVM from exiting when there was an issue...